assets/src/edit-story/theme.js

```diff
@@ -81,6 +85,7 @@ export function useTheme(selector) {
 }
 
 const theme = {
+  designSystemTheme,
```

Comment:
@barklund FYI I'm adding this into `theme` for being able to use the new colors from `design-system`, we can later replace it then. LMK if you had something else in mind.